packages/design-system/src/components/Button/style.scss

```diff
@@ -160,7 +187,16 @@
     width: 20px;
     display: inline-block;
     line-height: 1;
-    margin-right: 5px;
+
+    &-left {
+      margin-right: 5px;
+      order: -1;
+    }
+
+    &-right {
+      margin-left: 5px;
+      order: 1;
```

Comment:
you are smart :D I would do some stuff with flex-direction xD

Reply:
I tried and I gave up xD Then I realized that there's still `order` ^^